conf/ref_databases.config

```diff
@@ -10,194 +10,194 @@
 
 params {
     dada_ref_databases {
-        'coidb=221216' {
+        'coidb' {
             title = "COIDB - CO1 Taxonomy Database - Release 221216"
             file = [ "https://figshare.scilifelab.se/ndownloader/files/38787072", "https://figshare.scilifelab.se/ndownloader/files/38787069" ]
             citation = "Sundh J, Manoharan L, Iwaszkiewicz-Eggebrecht E, Miraldo A, Andersson A, Ronquist F. COI reference sequences from BOLD DB. doi: https://doi.org/10.17044/scilifelab.20514192.v2"
             fmtscript = "taxref_reformat_coidb.sh"
             taxlevels = "Kingdom,Phylum,Class,Order,Family,Genus,Species"
         }
-        'coidb' {
+        'coidb=221216' {
             title = "COIDB - CO1 Taxonomy Database - Release 221216"
             file = [ "https://figshare.scilifelab.se/ndownloader/files/38787072", "https://figshare.scilifelab.se/ndownloader/files/38787069" ]
             citation = "Sundh J, Manoharan L, Iwaszkiewicz-Eggebrecht E, Miraldo A, Andersson A, Ronquist F. COI reference sequences from BOLD DB. doi: https://doi.org/10.17044/scilifelab.20514192.v2"
             fmtscript = "taxref_reformat_coidb.sh"
             taxlevels = "Kingdom,Phylum,Class,Order,Family,Genus,Species"
         }
-        'midori2-co1=gb250' {
+        'midori2-co1' {
             title = "MIDORI2 - CO1 Taxonomy Database - Release GB250"
             file = [ "http://reference-midori.info/download/Databases/GenBank250/DADA2_sp/uniq/MIDORI2_UNIQ_NUC_SP_GB250_CO1_DADA2.fasta.gz" ]
             citation = "Machida RJ, Leray M, Ho SL, Knowlton N. Metazoan mitochondrial gene sequence reference datasets for taxonomic assignment of environmental samples. Sci Data. 2017 Mar 14;4:170027. doi: 10.1038/sdata.2017.27. PMID: 28291235; PMCID: PMC5349245."
             fmtscript = "taxref_reformat_midori2.sh"
-            dbversion = 'midori2-co1=gb250'
+            dbversion = 'MIDORI2-CO1 GB250 (http://reference-midori.info/download/Databases/GenBank250/DADA2_sp/uniq/MIDORI2_UNIQ_NUC_SP_GB250_CO1_DADA2.fasta.gz)'
```

Comment:
Theoretically I am fine with longer strings here, but the same info (links to files) is already given in the `file` field. I am not very fond of repeated information. If a different solution could be found, e.g. concatenate `dbversion` & `file` field in sbdi-export, then I would like that more.

Reply:
For this database, yes the info is the same as dbversion+file. But for e.g. UNITE or SBDI-GTDB we instead of file have a link to a page with an overview of the database / database version. I could not find a page like this for all databases, and then used the link to the actual file instead as SBDI wants a link for all databases. 
We can of course add another string for SBDI export if you like that better, or maybe there is some other suggestion?